tests/e2e/UsingBluemix.md

```diff
@@ -0,0 +1,95 @@
+# Run E2E tests on Bluemix
+
+This setup helps you run e2e tests with your local codes against a k8s cluster on BLuemix.
+
+* [Step 0: Install and configure IBM Cloud Kubernetes Service CLI](#step-0-install-and-configure-ibm-cloud-kubernetes-service-cli)
```

Comment:
Since there are only 4 steps, I wonder if this is needed (same about the GKE doc actually) =)

Reply:
IBM Cloud CLI `bx` is not that famous as `gcloud`. I added this step to help people point to the correct page. Looks like no one complains about the absence of gcloud installation in GKE doc?